cypress/support/step-definitions/common-steps.js

```diff
@@ -195,6 +195,10 @@ When('I open component preview', () => {
   commonButtonPreview().click();
 });
 
+When('I open component preview in Docs', () => {
```

Comment:
please, remove this step. You don't use it anywhere

Reply:
👍